bcda/main.go

```diff
@@ -300,25 +295,16 @@ func setUpApp() *cli.App {
                         },
                         Action: func(c *cli.Context) error {
 
-                                if ttl == "" {
-                                        ttl = os.Getenv("JWT_EXPIRATION_DELTA")
-                                        if ttl == "" {
-                                                ttl = "72"
-                                        }
-                                }
-        			ttlInt, err := strconv.Atoi(ttl)
-			        if err != nil || ttlInt <= 0 {
-					return fmt.Errorf("invalid argument '%v' for --ttl; should be an integer > 0", ttl)
-        			}
-
 				// Get ACO by CMS ID (since GenerateClientCredentials interface expects a Client ID)
 				aco, err := auth.GetACOByCMSID(acoCMSID)
                                 if err != nil {
                                         return err
                                 }                               
 
                                 // Generate new credentials
-				creds, err := auth.GetProvider().GenerateClientCredentials(aco.UUID.String(), ttlInt)
+				// TODO: GenerateClientCredentials interface to be updated as part of BCDA-1304
+				//       Passing in a ttl of 0 for now, since it is not used. 
+				creds, err := auth.GetProvider().GenerateClientCredentials(aco.UUID.String(), 0)
```

Comment:
To use the GenerateClientCredentials method, the ACO must have previously registered for and received credentials, and their client_id must have a value in their ACO record. So, we should use the aco.client_id here, not the aco.UUID.String(). The Okta plugin will error out here if we pass in the aco.UUID.String() here, because client_id for Okta is not the same as aco.UUID.

Reply:
In that case, do we still have a use for an internal ACO UUID?

Reply:
Updated, thanks DJ.